client/state/sites/domains/test/selectors.js

```diff
@@ -58,43 +57,31 @@ describe( 'selectors', () => {
 	} );
 
 	describe( '#decorateSiteDomains()', () => {
-		test( 'should return decorated site domains with autoRenewalMoment', () => {
-			const state = getStateInstance(),
-				domains = getDomainsBySiteId( state, firstSiteId );
+		test( 'should return decorated site domains with autoRenewalDate', () => {
+			const state = getStateInstance();
+			const domains = getDomainsBySiteId( state, firstSiteId );
 
 			const decoratedDomains = getDecoratedSiteDomains( state, firstSiteId );
 
-			const domainAutoRenewalMoment = moment( domains[ 0 ].autoRenewalDate );
-
-			expect( decoratedDomains[ 0 ].autoRenewalMoment.date() ).to.equal(
-				domainAutoRenewalMoment.date()
-			);
+			expect( decoratedDomains[ 0 ].autoRenewalDate ).to.equal( domains[ 0 ].autoRenewalDate );
```

Comment:
These tests no longer test anything: the selector returns the data from state without any transformation or memoization.

Reply:
Removed all the tests for `getDecoratedSiteDomains`, as it was removed.